### PR TITLE
Updating contour, stern, cloudformation, and k8s controller runtime versions

### PR DESCRIPTION
**Pull request by ytimocin**
# Description
Updating contour, stern, cloudformation, and k8s controller runtime versions

## Type of change
- This pull request is a minor refactor, code cleanup, test improvement, or other maintenance task and doesn't change the functionality of Radius (issue link optional).

### CODE REVIEW

File: pkg/cli/kubernetes/logstream/types.go [hunk 1]
```diff
@@ -32,6 +34,9 @@ type Options struct {
 	// KubeContext is the name of the kubernetes context to use for connection.
 	KubeContext string
 
+	// KubeClient is the kubernetes client to use for connection.
```
Comment (rynowak) [inline pkg/cli/kubernetes/logstream/types.go:37]: If `KubeContext` is unused can we remove it?

File: pkg/controller/service.go [hunk 2]
```diff
@@ -71,10 +72,11 @@ func (s *Service) Run(ctx context.Context) error {
 	port := s.Options.Config.Server.Port
 	healthProbePort := *s.Options.Config.WorkerServer.Port
 	mgr, err := ctrl.NewManager(s.Options.K8sConfig, ctrl.Options{
-		Logger:                 logger,
-		CertDir:                s.TLSCertDir,
```
Comment (rynowak) [inline pkg/controller/service.go:75]: So, do you know how the certificates should be configured now that this option has changed? The certificates are needed outside of test scenarios.

**Review by rynowak — CHANGES_REQUESTED** [review]
Overall this looks right, but there are two things to double check.

- Can we remove `KubeContext` from the options if it's unused?
- How do we configure certificates?

File: pkg/controller/service.go (hunk 2 above)
Comment (ytimocin) [reply pkg/controller/service.go:75]: There is also this setting: https://github.com/radius-project/radius/pull/7498/files#diff-b31c03e05358c6faab6e77c96909e14e14eeb0dc09412ede3329625adc197fa9R85. Can we sync offline on this one? Trying to understand where else we can need this property.

File: pkg/cli/kubernetes/logstream/types.go (hunk 1 above)
Comment (ytimocin) [reply pkg/cli/kubernetes/logstream/types.go:34]: Removed the **KubeContext**.

File: pkg/recipes/driver/terraform.go [hunk 3]
```diff
@@ -152,7 +152,7 @@ func (d *terraformDriver) Delete(ctx context.Context, opts DeleteOptions) error
 // prepareRecipeResponse populates the recipe response from the module output named "result" and the
 // resources deployed by the Terraform module. The outputs and resources are retrieved from the input Terraform JSON state.
 func (d *terraformDriver) prepareRecipeResponse(ctx context.Context, definition recipes.EnvironmentDefinition, tfState *tfjson.State) (*recipes.RecipeOutput, error) {
-	if tfState == nil || (*tfState == tfjson.State{}) {
+	if tfState == nil || reflect.DeepEqual(*tfState, tfjson.State{}) {
```
Comment (ytimocin) [inline pkg/recipes/driver/terraform.go:155]: This branch will be rebased after #7497 gets merged in.

**Review by rynowak — APPROVED** [review]

**Review by kachawla — APPROVED** [review]

**Pull request merged**